sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -399,24 +397,15 @@ def process_clear_timer(self):
       assert_that(actual, equal_to(expected))
 
   def test_pardo_state_timers(self):
```

Comment:
Why did this file need to change?

Reply:
That was a mistake, thanks for catching it.

Reply:
Yup, I'm also prone to these kinds of errors.